webui/react/src/components/AuthToken.tsx

```diff
@@ -9,6 +9,7 @@ import { copyToClipboard } from 'utils/dom';
 import css from './AuthToken.module.scss';
 
 const AuthToken: React.FC = () => {
+  // TODO update to use globalStorage.
```

Comment:
**question:** should this be done as part of this PR?

Reply:
yeah..for our ee repo